create-env.js

```diff
@@ -1,15 +1,15 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
 const fs = require('fs');
 const envVars = `
 # these are only generated on netlify deploy
```

Comment:
If we move to Vercel with this PR, we should remove this file. And thinking about it, probably everything that is Netlify-related...

Reply:
Agreed! I kept it here as it wasn't yet clear if we would deploy that on Netlify or not. I'll remove it!